src/main/java/axoloti/utils/Preferences.java

```diff
@@ -200,6 +208,37 @@ public String getCurrentFileDirectory() {
         return CurrentFileDirectory;
     }
 
+    public IPlatform getPlatform(Patch p) {
+        IPlatform target = null;
+        if (Platform != null && !Platform.isEmpty()) {
+            try {
+                Logger.getLogger(Preferences.class
+                        .getName()).log(Level.INFO, "Attempting to load target {0}", Platform);
+                ClassLoader cl = Axoloti.class.getClassLoader();
+                Class c = cl.loadClass(Platform);
+                Object o = c.newInstance();
```

Comment:
>Again, this seems like a highly unlikely scenario, distributing a separate platform JAR I mean. We're just overengineering in the present for something that may not exist in the future

I agree we should avoid over engineering, but in this case, I believe its a useful feature, without any real overhead.

dynamic class loading is a standard and useful feature of java, so I see no reason to start hard coding things that do not need to be.